QUsingStack.java

```diff
@@ -0,0 +1,61 @@
+class MyQueue {
+
+    /** Initialize your data structure here. */
+    Stack<Integer> lifo;
```

Comment:
Please add the other problem. Also please add comments explaining your approach and time and space complexities.